packages/gutenframe/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
It's a good idea to pull in `.eslintrc.js` to your package root like this:

https://github.com/Automattic/wp-calypso/blob/bd88f8fafad6505ef3783f959fd64fb5f0365b9a/packages/calypso-build/.eslintrc.js#L1-L5

Reply:
@sirreal should that be override in the root for `/packages/*`? Seems like every package has this. `__dirname` might make it tricky tho 🤔

Reply:
Doesn't seem possible to dynamically configure the rule for multiple directories. I'd love to be proven wrong 🙂

Reply:
Added in 2cdc03c.